config.ts

```diff
@@ -0,0 +1,3 @@
+export const config : {interval:number} = {
+    interval: Number(process.env.INTERVAL),
```

Comment:
- Use indentation of 2 instead of 4
- You can omit the return type, as Typescript infers it automatically
- It's better for this object to be default export, instead of named export, as we probably won't add any other exports to this file